apps/arweave/src/ar_config.erl

```diff
@@ -721,6 +721,144 @@ parse_options([{<<"shutdown_tcp_connection_timeout">>, Delay} | Rest], Config)
 		parse_options(Rest, NewConfig);
 parse_options([{<<"shutdown_tcp_connection_timeout">>, InvalidValue} | Rest], Config) ->
 	{error, {bad_type, shutdown_tcp_connection_timeout, integer}, InvalidValue};
+parse_options([{<<"shutdown_tcp_mode">>, Mode}|Rest], Config) ->
+	case Mode of
+		<<"soft">> ->
+			NewConfig = Config#config{ shutdown_tcp_mode = soft },
+			parse_options(Rest, NewConfig);
+		<<"hard">> ->
+			NewConfig = Config#config{ shutdown_tcp_mode = hard },
+			parse_options(Rest, NewConfig);
+		Mode ->
+			{error, {bad_value, shutdown_tcp_mode}, Mode}
+	end;
+
+%% Gun client parameters
+parse_options([{<<"http_client.http.keepalive">>, Timeout}|Rest], Config) ->
+	case Timeout of
+		<<"infinity">> ->
+			parse_options(Rest, Config#config{ 'http_client.http.keepalive' = infinity });
```

Comment:
FYI: The cli values for this option are true/false vs. here they are infinity/integer.